inc/common/purge.php

```diff
@@ -527,3 +527,19 @@ function rocket_clean_cache_theme_update( $wp_upgrader, $hook_extra ) {
 
 	rocket_clean_domain();
 }
+
+/**
+ * Purge WP Rocket cache on Slug / Permalink change.
+ *
+ * @since  3.4.2
+ * @author Soponar Cristina
+ *
+ * @param int   $post_id   The post ID.
+ * @param array $post_data Array of unslashed post data.
+ */
+function rocket_clean_cache_on_slug_change( $post_id, $post_data) {
+	if ( get_post_field( 'post_name', $post_id ) != $post_data['post_name'] ) {
```

Comment:
```suggestion
	if ( get_post_field( 'post_name', $post_id ) !== $post_data['post_name'] ) {
```

Per our coding standards, we always use strict comparison